src/lib/Form/Data/Location/LocationAssignSubtreeData.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\Location;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\Section;
+
+class LocationAssignSubtreeData
+{
+    /** @var \eZ\Publish\API\Repository\Values\Content\Section|null */
+    protected $section;
+
+    /** @var \eZ\Publish\API\Repository\Values\Content\Location|null */
+    protected $location;
+
+    public function __construct(?Section $section = null, ?Location $location = null)
```

Comment:
When those can be null?

Reply:
https://github.com/adamwojs/ezplatform-admin-ui/blob/1db795bf325b8ed65176bf12206f49239ebb7942/src/bundle/Controller/LocationController.php#L616

Reply:
I understand that at point of creating, but what would happend if you pass null explicite into this, and then handle form? Shouldn't this have also have some nonEmpty validation?